src/imageLoader/decodeImageFrame.js

```diff
@@ -1,10 +1,12 @@
 import { getOptions } from './internal/options.js';
 import webWorkerManager from './webWorkerManager.js';
 import decodeJPEGBaseline8BitColor from './decodeJPEGBaseline8BitColor.js';
-import { default as decodeImageFrameHandler } from '../shared/decodeImageFrame.js';
-import calculateMinMax from '../shared/calculateMinMax.js';
-import { initializeJPEG2000 } from '../shared/decoders/decodeJPEG2000.js';
-import { initializeJPEGLS } from '../shared/decoders/decodeJPEGLS.js';
+
+// TODO: Find a way to allow useWebWorkers: false that doesn't make the main bundle huge
+//import { default as decodeImageFrameHandler } from '../shared/decodeImageFrame.js';
+//import calculateMinMax from '../shared/calculateMinMax.js';
+//import { initializeJPEG2000 } from '../shared/decoders/decodeJPEG2000.js';
+//import { initializeJPEGLS } from '../shared/decoders/decodeJPEGLS.js';
```

Comment:
For script tag usage, bundle them all up. If we have complaints, we could always optimize it later by adding better configuration?

For fancier folk: https://webpack.js.org/guides/code-splitting/#dynamic-imports
Import them dynamically, and only when they're first used?
IE. async load the dependency when it's needed.

Reply:
Looks like I am also running out of memory when trying to build/test the library if I keep these lines in.

Reply:
Switched to exports-loader and it's almost all working properly. The only one that appears to be broken is jpegLossless.